source/documentation/install-guide/chap-System_Requirements.html.md

```diff
@@ -62,13 +62,13 @@ Browser support is divided into tiers:
 
 **Browser Requirements**
 
-| Support Tier | Operating System Family | Browser | Portal Access |
+| Support Tier | Operating System Family | Browser |
```

Comment:
I'm not sure what 'support tier' mean in oVirt.

Reply:
Something like "how much do we recommend to use it" :) But I agree it does not make sense in oVirt to call it like this. When I think of it a bit more I tend to agree with @gregsheremeta to have a statement similar to https://www.ovirt.org/download/  - even though Im not really sure the "We validate against and fully support " statement is correct. Maybe something like:
"We recommend to use one of the recent versions of the following "evergreen" browsers:..."?

Reply:
> Im not really sure the "We validate against and fully support " statement is correct

Yeah, fair enough :) I'd love to get there with the automated testing I'm working on, though :D